Unity3D/Assets/RosSharp/Editor/UrdfImporter/UrdfRobotExtensions.cs

```diff
@@ -59,5 +78,93 @@ public static void setKinematic(GameObject robot, bool isKinematic)
             foreach (Rigidbody rigidbody in rigidbodies)
                 rigidbody.isKinematic = isKinematic;
         }
+
+        public static bool ImportUrdfToUnity(Robot robot)
+        {
+            // Create URDF folder
+            if (!AssetDatabase.IsValidFolder("Assets/Urdf"))
+            {
+                AssetDatabase.CreateFolder("Assets", "Urdf");
+            }
+
+            // Create folder for URDF
+            if (!AssetDatabase.IsValidFolder("Assets/Urdf/" + robot.name))
+            {
+                AssetDatabase.CreateFolder("Assets/Urdf", robot.name);
+            }
+            // TODO: if folder exists, check if robot should be updated.
+
+            // Create folder for meshes
+            if (!AssetDatabase.IsValidFolder("Assets/Urdf/" + robot.name + "/meshes"))
```

Comment:
Please no dedicated mesh directory for a robot meshes.
If we have in our robot two meshes with the same file name located in different package folders, one mesh will be overwritten.
We shopuld maintain the original package structure from ROS

Reply:
I don't follow how the file will be overwritten, each imported URDF will have it's own meshes folder
`Assets/<robot_package>/meshes`